### PR TITLE
Fix issue with calculated width on small screens

### PR DESCRIPTION
**Pull request by txsmith**
This
screenshot from 2016-05-28 21 09 00
really really sucks when you use full-HD in portrait mode or any small screen.

Luckily the fix was easy :)

### CODE REVIEW

File: src/projects/project-overview-page.html [hunk 1]
```diff
@@ -56,9 +56,11 @@
         padding: 0 20px;
         height: 100%;
       }
+      .info-area,
       iron-pages {
-        width: 75%;
+        max-width: 1100px;
```
Comment (txsmith) [inline src/projects/project-overview-page.html:61]: This is to make sure that views do not get to wide on full-HD screens

**Comment by TimvdLippe** [comment]
Could you include screenshots for smaller sized screens? Also the changes seem duplicate. Since they should be styled in a parent, should we put this in rite-app instead?

File: src/projects/project-overview-page.html (hunk 1 above)
Comment (TimvdLippe) [inline src/projects/project-overview-page.html:61]: Please leave that as inline comment too.

**Comment by txsmith** [comment]
The change is duplicate to prevent the scrollbar from appearing in the middle of your screen. If you want to see how it works, just make your browser window smaller.

**Comment by TimvdLippe** [comment]
LGTM. Could not test for the full-HD size, since I do not have such a screen :smile:

**Pull request merged**